salt/orch/removal.sls

```diff
@@ -340,7 +340,7 @@ shutdown-target:
 # (it will appear as "unaccepted")
 remove-target-salt-key:
   salt.wheel:
-    - name: key.delete
+    - name: key.reject
```

Comment:
For this to work you need to include `include_accepted: True`.

Reply:
Right. Fixed.